frontend/app/component/form/polling_station_choice/PollingStationChoiceForm.tsx

```diff
@@ -15,9 +15,11 @@ export function PollingStationChoiceForm() {
 
   const handleSubmit = () => {
     const parsedStationNumber = parseInt(pollingStationNumber, 10);
-    if (pollingStations.some((ps) => ps.number === parsedStationNumber)) {
-      navigate(`./${pollingStationNumber}/recounted`);
-    }
+    pollingStations.forEach((pollingStation) => {
```

Comment:
Why changing to a `forEach`? isn't `some` better conveying what is happening here?

Reply:
I couldn't use the ps in the navigate when I used some. If this is possible please teach me 🤓

Reply:
`some` doesn't return the polling station, so you can't get the id from it.

Maybe using [`find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) would be better than a `forEach`.

Reply:
Ah, I see! `find` could indeed work. Not tested, but maybe something like this:
```
const pollingStation = pollingStations.find((pollingStation) => pollingStation.number === parsedStationNumber);
if (pollingStation) {
   navigate(`./${pollingStation.id}/recounted`);
}
```

Reply:
Changed in 35bb4e286e598bb6264d02f2c3380180a0b48e60